eagle-core/eagle-policy/eagle-policy-base/pom.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0"?>
```

Comment:
It's good to separate policy engine out for reusing in both alerting and aggregation.